frontend/src/scenes/dashboard/DashboardHeader.js

```diff
@@ -69,16 +73,25 @@ export function DashboardHeader({ logic }) {
                                 </Tooltip>
                             ) : null}
 
+                            <Tooltip title={'Share dashboard.'}>
+                                <Button
+                                    className="button-box-when-small enable-dragging-button"
+                                    type={dashboard.is_shared ? 'primary' : ''}
+                                    onClick={() => setShowShareModal(true)}
+                                    data-attr="dashboard-share-button"
+                                >
+                                    <ShareAltOutlined />
+                                    {dashboard.is_shared ? 'Shared' : 'Share dashboard'}
+                                </Button>
+                            </Tooltip>
+
                             <Tooltip title={'Click here or long press on a panel to rearrange the dashboard.'}>
                                 <Button
                                     className="button-box-when-small enable-dragging-button"
                                     type={draggingEnabled === 'off' ? 'primary' : ''}
                                     onClick={draggingEnabled === 'off' ? enableDragging : disableDragging}
                                 >
                                     {draggingEnabled !== 'off' ? <UnlockOutlined /> : <LockOutlined />}
-                                    <span className="hide-when-small">
-                                        {draggingEnabled !== 'off' ? 'Lock Dragging' : 'Dragging Locked'}
```

Comment:
The tooltip for the lock is misleading without the "lock dragging" label. It's unlocked, yet the tooltip implies that it's currently locked.

![image](https://user-images.githubusercontent.com/53387/86570356-15ba6180-bf70-11ea-89dd-b3ed2ff1bfed.png)

If you'd like to remove the text completely, it's also best to give the button the square shape